src/wallet/api/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ target_link_libraries(wallet_api
     extra)
 
 set_property(TARGET wallet_api PROPERTY EXCLUDE_FROM_ALL TRUE)
+target_compile_options(wallet_api PRIVATE -fvisibility=default)
```

Comment:
`-fvisibility=default` was not the only thing here that mattered: there was also `__attribute__((used))`, which prevents the compiler/linker from dropping unused functions (which caused some linking errors at one point, IIRC).

Reply:
yeah damn, attribute(used) looks like its for the linker so its completely separate.